src/main/java/org/opentripplanner/model/plan/ScheduledTransitLeg.java

```diff
@@ -313,6 +317,31 @@ public void addAlert(TransitAlert alert) {
     transitAlerts.add(alert);
   }
 
+  @Override
+  @Nullable
+  public Float accessibilityScore() {
+    return accessibilityScore;
+  }
+
+  public ScheduledTransitLeg withAccessibilityScore(Float score) {
```

Comment:
Should we override this for frequency leg?

Reply:
Yes, I added it.